libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_subscription.c

```diff
@@ -224,7 +226,7 @@ static bool _waitForCount( IotMutex_t * pMutex,
  * @brief Wait for a reference count to reach a target value, using FreeRTOS semaphores,
  * subject to a timeout.
  */
-static bool _waitForCountSemaphore( StaticSemaphore_t * pSem,
+static bool _waitForCountSemaphore( SemaphoreHandle_t * pSem,
```

Comment:
`SemaphoreHandle_t sem`?

Reply:
It can be Semaphore Handle itself. I will do it as a follow up PR.